core/src/actions/properties.ts

```diff
@@ -259,7 +259,7 @@ export class PropertyView extends AuthenticatedAction {
 
   async runWithinTransaction({ params }) {
     const property = await Property.findById(params.id);
-    const source = await property.$get("source");
+    const source = await property.$get("source", { scope: null });
```

Comment:
👍 makes sense that if you are calling a property explicitly by It's ID, you can see it in any state